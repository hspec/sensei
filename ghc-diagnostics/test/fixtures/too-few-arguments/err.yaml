version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/too-few-arguments/Foo.hs
  start:
    line: 3
    column: 7
  end:
    line: 3
    column: 11
severity: Error
code: 83865
message:
- |-
  Couldn't match expected type: Maybe Int
              with actual type: a0 -> Maybe a0
- |-
  Probable cause: `Just' is applied to too few arguments
  In the expression: Just
  In an equation for `foo': foo = Just
hints: []
