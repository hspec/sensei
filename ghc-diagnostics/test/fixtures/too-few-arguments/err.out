test/fixtures/too-few-arguments/Foo.hs:3:7: error: [GHC-83865]
    • Couldn't match expected type: Maybe Int
                  with actual type: a0 -> Maybe a0
    • Probable cause: `Just' is applied to too few arguments
      In the expression: Just
      In an equation for `foo': foo = Just

